CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.4.0 (--)
+- Support full height dashboards and scaled thumbnails [#312](https://github.com/grafana/grafana-image-renderer/pull/312)
```

Comment:
@AgnesToulet what is the process for documenting the changes here?

The behavior changes are:
1. negative scale will create a scaled down thumbnail
2. height=-1 will create an image as tall as it needs to be based on the webpage size

Reply:
Sounds good @AgnesToulet. Let me know if you need any help.